sound/soc/sof/intel/hda-stream.c

```diff
@@ -525,6 +525,16 @@ int hda_dsp_stream_init(struct snd_sof_dev *sdev)
 		return -ENOMEM;
 	}
 
+	if (sdev->hbus) {
```

Comment:
I thought we needed different ring buffers for CORB and RIRB, similar to capture and playback?

Reply:
they are allocated together(4KB) and corb/rirb are initialized to use different regions( pls refer to snd_hdac_bus_init_cmd_io()):
bus->corb.addr = bus->rb.addr;
bus->rirb.addr = bus->rb.addr + 2048;